test/nn/test_pooling.py

```diff
@@ -1209,9 +1209,9 @@ def expected_output(dim, dtype):
                 return torch.stack([col, col + 2], 1).view(2, 2, 2, 2)
 
         if adaptive:
-            cls_name = 'AdaptiveMaxPool{}d'.format(num_dim)
+            cls_name = f'AdaptiveMaxPool{num_dim}d'
         else:
-            cls_name = 'MaxPool{}d'.format(num_dim)
+            cls_name = f'MaxPool{num_dim}d'
```

Comment:
Could this change in formatting make the formatting behavior differently?

Reply:
So I tested on linux and could not reproduce either. I am going to revert this line and see what happens on CI

Reply:
So it doesn't fail anymore. I can't think of a reason why it's happening. Created https://github.com/pytorch/pytorch/issues/105716 to track